test/spot-checker.js

```diff
@@ -64,9 +97,18 @@ test('fetchCAR fails with statusCode=701 (unsupported host type)', async () => {
     protocol: 'http',
     address: '/ip99/1.2.3.4.5/tcp/80/http',
     cid: KNOWN_CID,
-    stats
+    stats,
+    dagScope: 'all'
+  })
+  await spark.fetchCAR({
+    protocol: 'http',
+    address: '/ip99/1.2.3.4.5/tcp/80/http',
+    cid: KNOWN_CID,
+    stats,
+    dagScope: 'block'
   })
-  assertEquals(stats.statusCode, 701, 'stats.statusCode')
+  assertEquals(stats.fullRetrieval.statusCode, 701, 'stats.statusCode')
+  assertEquals(stats.blockRetrieval.statusCode, 701, 'stats.statusCode')
```

Comment:
Please update the assertion message `stats.statusCode` to include `fullRetrieval` or `blockRetrieval`. The same comment applies to all other similar places below.

Reply:
Resolved https://github.com/CheckerNetwork/spark-spot-check/pull/6/commits/7c14d8abf0f44b26c7daea75fc83ea20953a6412